src/ospgrillage/postprocessing.py

```diff
@@ -17,7 +17,6 @@
 from ospgrillage.load import ShapeFunction
 from ospgrillage.static import solve_zeta_eta
 
-import openseespyvis.Get_Rendering as opsplt
```

Comment:
Can't delete this variable; it is used throughout codebase. #74 redfines it with vfo which is now accepted. Pull again and check it works.